gprofiler/profilers/java.py

```diff
@@ -537,6 +541,8 @@ def __init__(
         self._java_safemode = java_safemode
         if self._java_safemode:
             logger.debug("Java safemode enabled")
+        self._profiled_processes: Set[Process] = set()
```

Comment:
@YishaiZinkin 's PR adds something similar. Whomever gets merged first - the latter needs to base on the new code :sweat_smile: